### PR TITLE
styling fixes for quickbook docs

### PR DESCRIPTION
**Pull request by cdw9**
https://github.com/boostorg/boostlook/issues/54

### CODE REVIEW

**Review by julioest — COMMENTED** [review]
Looks great so far! 

A couple of things to address:
- adjust the large horizontal spacing between code blocks, tables, and other elements
- update the spirit nav icons to match Learn Docs
- align the `.title` with the spirit nav

**Review by julioest — CHANGES_REQUESTED** [review]
Awesome! Looks much better

Only thing left is updating spirit nav icons to match Antora.

Lmk if you need any help!

File: boostlook.css [hunk 1]
```diff
@@ -499,15 +503,14 @@ p, h1, h2, h3, h4, h5, h6 {
...
+  margin: 1em 0;
   border: 1px solid var(--bl-table-border-color);
 }
 
 .boostlook:not(:has(.doc)) th {
   background-color: var(--bl-table-head-background);
   text-align: left;
   padding: 0.25em 0.55em;
-  font-weight: 550;
 }
```
Comment (rbbeeston) [inline boostlook.css:514]: Curious why the font-weight was removed.

**Review by rbbeeston — COMMENTED** [review]
one comment, but ok to merge once Julio's items are addressed.

File: boostlook.css (hunk 1 above)
Comment (cdw9) [reply boostlook.css:514]: I removed this so the default weight of 700 will display

**Comment by julioest** [comment]
Hey @cdw9 - it seems that the pre-commit hook didn't run based on the CI job.

Run a `pre-commit install` locally, so next time the hook catches and fixes the errors in the CSS file. Thanks!

**Pull request merged**